### PR TITLE
Inline to less

### PR DESCRIPTION
**Pull request by florapdx**
#### What's this PR do?
Changes over from inline styles to external styles using Less.

#### Where should the reviewer start?
package.json/gulpfile, then check my work in all less files

#### How should this be manually tested?
Load up example files and/or load up XPM in Sprintly and npm link to your local copy of sprintly-ui in this branch.

#### Any background context you want to provide?
Change is being made due to perf concerns around inline styles when many components are rendered at the same time

#### What ticket or issue # does this fix?
Fixes #9302 in Sprintly

#### Screenshots (if appropriate)
n/a

#### Definition of Done:
- [x] Does this contain breaking changes?
- [x] Did you bump the version number accordingly?
- [x] Is there appropriate test coverage?
- [x] Does this PR require a regression test? All fixes require a regression test.
- [x] Does this add new dependencies?
- [x] Will this feature require a new piece of infrastructure be implemented?
- [x] Did you update the documentation per these changes?

### CODE REVIEW

**Comment by florapdx** [comment]
Two thing that need to happen before this is merged:
-- use classSet()
-- add prepublish script so we don't run into build-errors when running npm install in other environs

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "sprintly-ui",
   "title": "Sprintly UI",
   "description": "A React-based UI kit for building Sprintly interfaces",
-  "version": "0.3.2",
+  "version": "0.4.2",
```
Comment (justinabrahms) [inline package.json]: This is technically a backwards incompatible change, as people who had this before will now see broken styles.

File: src/components/estimator.js [hunk 2]
```diff
@@ -26,10 +25,10 @@ var Estimator = React.createClass({
     readOnly: React.PropTypes.bool,
     itemType: React.PropTypes.string.isRequired,
     score: React.PropTypes.string.isRequired,
-    estimateChanger: React.PropTypes.object.isRequired
+    estimateChanger: React.PropTypes.object
```
Comment (justinabrahms) [inline src/components/estimator.js]: As this is now not required, we might get an error on line 77 where we don't check to see if it exists before we call things on it?

File: src/components/tag_editor.js [hunk 3]
```diff
@@ -16,10 +15,10 @@ var TagEditor = React.createClass({
     modelId: React.PropTypes.arrayOf(React.PropTypes.number),
     readOnly: React.PropTypes.bool,
     tags: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
-    tagChanger: React.PropTypes.object.isRequired
+    tagChanger: React.PropTypes.object
```
Comment (justinabrahms) [inline src/components/tag_editor.js]: Same as before with the non-required notion and errors on lines 65 & 84.

**Pull request merged**